src/nft/components/details/detailsV2/__snapshots__/DataPageTable.test.tsx.snap

```diff
@@ -464,7 +464,7 @@ exports[`data page listings table content loads with a given asset 1`] = `
             <div
               class="c10 css-18hn7mq"
             >
-              5 months
+              4 months
```

Comment:
weird i had this too, and then in a later test it wanted 5 again, seems indeterministic?

Reply:
I let @cbachmeier know about this and I think he merged a fix for it yesterday. It's deterministic, but based on an offset from a fixed timestamp. He updated the test to account for this.